src/cg/cst_cg_map.c

```diff
@@ -368,12 +370,12 @@ void cst_read_voice_feature(cst_file fd, char **fname, char **fval)
     *fval = cst_read_padded(fd, &temp);
 }
 
-int cst_read_int(cst_file fd)
+int32_t cst_read_int(cst_file fd)
 {
```

Comment:
Since we use `int16_t` and `int_32t` maybe the function should be renamed `cst_read_int32`